build.sh

```diff
@@ -35,7 +35,7 @@ while [[ $# > 0 ]]; do
             args=( "${args[@]/$1}" )
             args=( "${args[@]/$2}" )
             shift
-            ;;        
+            ;;
```

Comment:
Yes, my editor trims whitespace. Which works fine when I'm creating a file, but less fine when I'm not.